### PR TITLE
Add Netlify as a destination for terminal deployments

### PR DESCRIPTION
**Pull request by kenny-io**
Netlify allows users to deploy Gridsome sites right from the terminal. This PR adds Netlify to the list of terminal deployment destinations in the Gridsome documentation site and also adds the terminal deployment instructions in the `/deploy-to-netlify/` page on the docs as requested in #592

### CODE REVIEW

File: docs/deploy-to-netlify.md [hunk 1]
```diff
@@ -2,24 +2,62 @@
...
+Netlify also allows you to deploy Gridsome applications directly from the terminal.
+
+Install the Netlify CLI.
+
+```bash
+npm install netlify-cli -g
+
+```
+
+Login into your Netlify account via the CLI.
```
Comment (TerabyteTiger) [inline docs/deploy-to-netlify.md:40]: I'm not sure if one of these versions is grammatically more correct, but this should be either: 

- "Log into" or
- "Log in to"

**Review by TerabyteTiger — COMMENTED** [review]
Just a small grammar fix :) 

Otherwise looks good 👍🏻

File: docs/deploy-to-netlify.md (hunk 1 above)
Comment (kenny-io) [reply docs/deploy-to-netlify.md]: Thank you for the quick review @TerabyteTiger I just pushed a fix for the typo!

**Review by TerabyteTiger — APPROVED** [review]
Looks good to me 👍🏻

**Comment by TerabyteTiger** [comment]
I'm going to let thing hang so that hopefully someone else can take a second look before merging - but if nobody responds in 2 days feel free to tag me again and I'll merge it then :) 

Thanks for your contribution!

**Comment by thetre97** [comment]
LGTM!

**Pull request merged**